starter-code/client/src/pages/Ride.js

```diff
@@ -450,28 +501,98 @@ export default function Ride() {
             <p className="error-msg">{errorMsgs.seats_available}</p>
           )}
           <div>
-            <input
-              type="checkbox"
-              name="pickup"
-              checked={shouldPickup}
-              onChange={handlePickupChange}
-              className="form-field"
-            />{" "}
-            Driver pick up?
+            Are you willing to pickup riders?
+            <div className="radio-btn">
+              <div>
+                <input
+                  type="radio"
+                  name="should_pickup"
+                  value="yes"
+                  checked={shouldPickupRiders}
+                  onChange={handlePickupRadioBtnChange}
+                />{" "}
+                Yes
+              </div>
+              <div>
+                <input
+                  type="radio"
+                  name="should_pickup"
+                  value="no"
+                  checked={!shouldPickupRiders}
+                  onChange={handlePickupRadioBtnChange}
+                />{" "}
+                No
+              </div>
+            </div>
           </div>
-          <input
-            type="number"
-            onChange={handleInputChange}
-            value={values.rider_radius}
-            className="form-field"
-            placeholder="Rider Radius"
-            name="rider_radius"
-            autoComplete="off"
-            disabled={!shouldPickup}
-          />
-          {errorMsgs.rider_radius && (
-            <p className="error-msg">{errorMsgs.rider_radius}</p>
+          {shouldPickupRiders && (
+            <div>
+              How far are you willing to pickup riders from your start location
+              (in meters)
+              <Tooltip title="1600 meters = 0.99 miles">
```

Comment:
Add question mark after statement. And you can probably just round the 0.99 miles to 1.

Reply:
done